packages/bsky/tests/views/mutes.test.ts

```diff
@@ -88,6 +88,20 @@ describe('mute views', () => {
     ).toBe(false)
   })
 
+  it('removes content from muted users on getListFeed', async () => {
+    const listRef = await sc.createList(bob, 'test list', 'curate')
+    await sc.addToList(alice, bob, listRef)
+    await sc.addToList(alice, carol, listRef)
+    await sc.addToList(alice, dan, listRef)
+    const res = await agent.api.app.bsky.feed.getListFeed(
+      { list: listRef.uriStr },
+      { headers: await network.serviceHeaders(alice) },
+    )
+    expect(
+      res.data.feed.some((post) => [bob, carol].includes(post.post.author.did)),
+    ).toBe(false)
+  })
+
```

Comment:
Looks like this got moved over to `mute-list.test.ts`, I expect we can nix it here.

Reply:
we want to test both mutes & mute lists

the list that's getting created here is the "curation" list not the mute list 👍